connectathon-scenarios-2019-09/README.md

```diff
@@ -0,0 +1,91 @@
+## Argonaut PAMA Project: September 2019 Connectathon
+
+We'll be participating in the CDS Hooks track of the September 2019
+Connectathon, testing our draft CDS Hooks for PAMA IG.  The track is designed
+for clients and servers that want to try out a full spectrum of integration
+from:
+
+1. Automated rating of orders in-workflow
+2. Suggestions for alternative orders presented in-workflow
+3. App-mediated order selection through SMART Web Messaging
+
+## Scenario 1. Automated rating of orders in-workflow
+
+* EHR provides an order-entry form that triggers the `order-select` hook as
+orders are entered (and optionally triggers `order-sign` when full order
+details are present). Order entry must be able to populate at least: patient,
+provider, and a `draftOrders` bundle containing a `ServiceRequest` with a
+`code` and a `reasonCode`. [See example request](./examples/scenario-01-outbound-request.json).
+
+For this connectathon scenario, we recommend that the order
+entry screen provides a way to easily enter any of the following three orders:
+
+|`ServiceRequest.code`|`ServiceRequest.reasonCode`|Expected PAMA rating|
```

Comment:
Since the IG says "Recommend SNOMED/LOINC if CPT is unavailable" shall we land on some sample SNOMED/LOINC codes that participants can plan to test with if they meet that criteria?

Reply:
I think the clinical narratives should spell this out, for sure. I don't know how helpful it is to call out all these cases specifically in our connectathon track -- but would object to calling out more.